pages/common/gradle.md

```diff
@@ -1,15 +1,27 @@
 # gradle
 
-> Gradle is the official build system for Android Studio.
+> Gradle is an advanced open source build automation system.
 
 - Compile a package:
 
 `gradle build`
 
+- Exclude test task:
+
+`gradle build -x test`
+
+- Run in offline mode:
```

Comment:
Can you expand the description a bit to explain what this offline mode is ? It is not immediately clear to me what it does.

Reply:
👍